libpod/container_config.go

```diff
@@ -120,6 +120,10 @@ type ContainerRootFSConfig struct {
 	// with the size specified in ShmSize and populate this with the path of
 	// said tmpfs.
 	ShmDir string `json:"ShmDir,omitempty"`
+	// ShmShare indicates whether /dev/shm can be shared with other containers
+	ShmShare bool `json:"ShmShare,omitempty"`
+	// ShmMount indicates whether a tmpfs should be created and mounted on  /dev/shm
+	ShmMount bool `json:"ShmMount,omitempty"`
```

Comment:
This could break old containers, which won't have this set to true - maybe `NoShmMount` instead?

Reply:
@mheon How about now?